packages/image_picker/android/build.gradle

```diff
@@ -37,8 +37,5 @@ android {
 }
 
 dependencies {
-    api 'com.github.bumptech.glide:glide:4.0.0'
```

Comment:
(This comment is about lines 19-21, but github doesn't allow me to comment there)

Can the jitpack.io repository be removed now that we removed the external library? I think we only added it to get the external library.